README.md

```diff
@@ -51,6 +52,13 @@ or the libraries locally installed:
 $ BOOST_ROOT=/path/to/boost/libraries make all
 ```
 
+#### Windows 
+
+The makefile will not work on Windows with MSVC. Not tested with neither CygWin nor MinGw. Execute the following instead:
+```
+python setup.py install boost-location=<path_to_your_boost>
+```
+
 Testing
 -------
 
```

Comment:
Please also add instructions on how to run the tests and build the documentation Windows.